### ISSUE TITLE
Warden home screen broken?

### ISSUE THREAD

**Issue by Jacksper13** [mynodebtc/mynode#613]
Just upgraded to latest version of Warden on the Applications page, c327e84fb95fa6e2, and run the command indicated in the Warden app info area:

![image](https://user-images.githubusercontent.com/63911988/147882088-0afaf7d6-b334-4847-9875-166a9b5f76e9.png)

After some random fun facts and stuff showing up in the terminal, I get something like "running on 192.168.XXX.XXX:5010" where XXX indicates the local mynode address. So I manually type this in the browser (which by the way only works with http and not https - ISSUE 1), and I get to the Warden main screen, pretty cool. I hit the "Next" button or get started or whatever it says in that first screen and I get this (ISSUE 2):
![image](https://user-images.githubusercontent.com/63911988/147882009-d7163228-c6c3-42c7-91d3-9062edb6b23b.png)

Then I can go to the Terminal and Settings tab, which I successfully see and I see the URL change to 192.168.XXX.XXX:5010/terminal or /settings, which is fine but whenever I click "Home" again it takes me to /terminal. I need to type the entire URL with /home to get to the broken main sceen again (ISSUE 3).

Also, if the idea is to eventually end up getting to 192.168.XXX.XXX:5010, shouldn't this be reflected in the Warden app info area? Or even better, instead of having an "info" area like Pyblock, have it be an actual button that says "Warden" (like say the Thunderhub app) that opens a new tab with the :5010 extension? (ISSUE 4)

Running on RPI4 with SSD, mynode version 2.49

**Comment by tehelsper** [comment]
Yeah, there does seem to be some issues with navigating the web UI, but if I click around it seems to load eventually. Also, accessing via tor doesn't seem to have the error issues.

I added HTTPS (missed it originally) but it has additional issues. Clicking the links will cause it to revert back to HTTP for some reason.

The web UI seemed less stable than the CLI window, so I didn't run it as a service to keep the focus on the more popular CLI interface.
